packages/extension-ui/src/Popup/index.stories.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+
+import { View } from '../components';
+import Popup from '.';
+
+const PopupWrapper = ({ initialPath }: { initialPath: string }) => (
+  <View>
+    <MemoryRouter initialEntries={[initialPath]}>
+      <Popup />
+    </MemoryRouter>
+  </View>
+);
+
+export default {
+  component: PopupWrapper,
+  parameters: {
+    layout: 'fullscreen'
+  }
+} satisfies Meta<typeof Popup>;
```

Comment:
Use the `decorators` option for any kind of wrappers around the components. We don't want to spoil the `component` property, as SB infers some data in some cases from whatever's passed to it. So:
```suggestion
export default {
  component: Popup,
  parameters: {
    layout: 'fullscreen'
  },
  decorators: [
    (Story) => (
      <View>
        <MemoryRouter initialEntries={[initialPath]}>
          <Story />
        </MemoryRouter>
      </View>
    )
  ]
} satisfies Meta<typeof Popup>;
```